cli/src/utilities/printOutput.ts

```diff
@@ -108,12 +126,21 @@ export async function printOutput(
     info(`3. Copy these keys and paste them into your .env file.`);
     info(`4. Optionally, follow one of these guides to get started with Firebase:`);
     highlight(`https://docs.expo.dev/guides/using-firebase/#next-steps`);
-    info(``);
+    if (cliResults.packages.some((pkg) => pkg.name === 'vexo-analytics')) {
+      printVexoSteps();
+    }
     success(`Once you're done, run the following to get started: `);
     info(``);
   } else {
-    success('Success! 🎉 Now, just run the following to get started: ');
-    info(``);
+    if (cliResults.packages.some((pkg) => pkg.name === 'vexo-analytics')) {
+      success(`Success! 🎉 Now, here's what's next:`);
+      info(``);
```

Comment:
is this `info(``)` necessary if `printVexoSteps()` already has the padding?

Reply:
This one is not necessary, Im removing it.